scripts/base/frameworks/telemetry/options.zeek

```diff
@@ -6,6 +6,10 @@ module Telemetry;
 # to see the options without needing the rest.
 
 export {
+	## Address used to make metric data available to Prometheus scrapers via
+	## HTTP.
+	const metrics_address: addr = 127.0.0.1 &redef;
```

Comment:
Tim I wonder whether we should switch this over to using the `ZEEK_DEFAULT_LISTEN_ADDRESS` environment variable, like [for Broker](https://github.com/zeek/zeek/blob/master/scripts/base/frameworks/broker/main.zeek#L27).

The benefit would be that anything that already works by adjusting that variable will transparently apply here, and it seems pretty likely to me that these will broadly match (if you run a cluster for which you want Broker to listen globally, you'll want that for telemetry too, and vice versa). As-is, I imagine a lot of cluster users will want to adjust it somewhere to listen globally.

Reply:
Ah yeah, that would be a good choice for the default. We were defaulting to localhost in the existing code, so I opted for that.